### ISSUE TITLE
Documentation would be nice

### ISSUE THREAD

**Issue by dented42** [tonyg/minimart#1]
It would be nice to have some scribble documentation available to make playing around easier.

**Comment by tonyg** [comment]
Yes, you're right. For now, does the [Marketplace documentation](http://tonyg.github.io/marketplace/) help at all?

I'm sorry I can't offer anything more Minimart-specific at the moment; I am working on designing an `actor` form for [Prospect](https://github.com/tonyg/prospect) that is similar to both the `actor` form of Minimart and the various macros of Marketplace.

**Comment by tonyg** [comment]
Incidentally, thanks for giving Minimart a try! And thanks for your feedback here. I'm very keen to hear about your experiences using the language, so please feel free to get in touch either here or via any of the media on my [contact page](http://homepages.kcbbs.gen.nz/tonyg/contact.html).
